cilium/cmd/endpoint_config.go

```diff
@@ -30,7 +30,7 @@ var listOptions bool
 var endpointConfigCmd = &cobra.Command{
 	Use:     "config <endpoint id> [<option>=(enable|disable) ...]",
 	Short:   "View & modify endpoint configuration",
-	Example: "endpoint config 5421 DropNotification=disable",
+	Example: "endpoint config 5421 DropNotification=false TraceNotification=false",
```

Comment:
As a user: What's the difference between the Trace and the Debug?

Reply:
Should be simple: Debug should be a developer only flag. We should remove it from the examples. Trace is for end users as well. The overhead of debug will be higher than the trace.

Reply:
I think the naming is confusing. The `CILIUM_NOTIFY_DBG_CAPTURE` messages are sent if `ENABLE_TRACE` is defined. That is currently always defined, and it's defined independently from `DEBUG`. `ENABLE_TRACE` can be undefined if the `--disable-trace` is given to Cilium. However, that option is used nowhere:
```
$ grep -r disable-trace *
daemon/main.go:		"disable-trace", false, "Disable ability to trace datapath actions")
```
Therefore, we currently always send the `CILIUM_NOTIFY_DBG_CAPTURE` messages, which I believe is a bug.

As @tgraf mentioned, the new TraceNotification is to enable a new set of messages, which are not only for debugging, but are as useful to users as drop messages.

I propose to remove the `ENABLE_TRACE` define, and use `DEBUG` instead to enable the `CILIUM_NOTIFY_DBG_CAPTURE` messages. I'll also remove the currently unused `--disable-trace` option.

Reply:
Pushed a new commit to replace `ENABLE_TRACE` with `DEBUG` and remove `disable-trace`.